components/AppServices/src/MultiTarget.props

```
@@ -4,6 +4,6 @@
           MultiTarget is a custom property that indicates which target a project is designed to be built for / run on.
           Used to create project references, generate solution files, enable/disable TargetFrameworks, and build nuget packages.
         -->
-        <MultiTarget>uwp;netstandard;</MultiTarget>
+        <MultiTarget>uwp;netstandard;net8.0-windows10.0.17763.0</MultiTarget>
```

@Arlodotexe not entirely sure about this, is this the right way to add a vanilla .NET 8 Windows TFM here?

---

```suggestion
        <MultiTarget>uwp;netstandard;</MultiTarget>
```

Like the comment says, our custom `MultiTarget` property is used to enable and disable the TargetFrameworks we support per-platform.

The TFM for modern .NET support on UWP is with the other UWP TFMs in our Toolkit Tooling repo on the [`uwp-net8-windows`](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/tree/uwp-net8-windows) branch. Majority of the changes here look fine, but you'll need that tooling branch.

---

Since the idea with this change is to add modern .NET support (TFM `net8.0-windows10*`) to AppServices wherever `uap` was used before (agnostic to uwp/wasdk/wpf), we'll continue to use the `MultiTarget` system for this.

We'll still have `HasWinUI` disabled, which downgrades the component to just the underlying TFM used, no WinUI. In the case of the `uwp` MultiTarget used for modern .NET, `UseUwp` and `UseUwpTools` [aren't enabled](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/blob/uwp-net8-windows/MultiTarget/MultiTargetIdentifiers.props#L63) when `HasWinUI` is set to false.